share/wake/lib/system/io.wake

```diff
@@ -130,20 +130,30 @@ export def read (path: Path): Result String Error =
         Fail f -> Fail (makeError f)
 
 target writeImp inputs mode path content =
+    # TODO: move this runner out of writeImp and don't directly depent on the parameters passed in to writeImp
+    #       See mkdirRunner for example of proper implementation
```

Comment:
Is this a TODO you're wanting to get in as part of this PR, or as a followup?
```suggestion
    # TODO: move this runner out of writeImp and don't directly depend on the parameters passed in to writeImp
    #       See mkdirRunner for example of proper implementation
```

Reply:
Either TBH. I'll probably kick to a different PR since this one is getting large